pkg/cmd/server/origin/run_components.go

```diff
@@ -376,7 +376,7 @@ func (c *MasterConfig) RunDeploymentImageChangeTriggerController() {
 // RunSDNController runs openshift-sdn if the said network plugin is provided
 func (c *MasterConfig) RunSDNController() {
 	oClient, kClient := c.SDNControllerClients()
-	controller, err := sdnfactory.NewMasterPlugin(c.Options.NetworkConfig.NetworkPluginName, oClient, kClient)
+	controller, err := sdnfactory.NewMasterPlugin(c.Options.NetworkConfig.NetworkPluginName, oClient, kClient, c.EtcdHelper)
```

Comment:
Currently etcd handler is used for 2 cases:
(a) Migration and (b) VNID range creation
Even if we make a workaround for (a), I don't know how we can deal with (b) without etcd handler. Check this line https://github.com/openshift/openshift-sdn/pull/303/files#diff-643475b06fe7640bd0eab5a38497f7beR40
What's the downside of keeping etcd access in controller? RunSecurityAllocationController() also uses etcd handler. Can we fix this in a separate pr to unblock dependent prs?
@smarterclayton @liggitt